### ISSUE TITLE
"sanity deploy" not working in CI/CD

### ISSUE THREAD

**Issue by paulThorsen** [sanity-io/sanity#2437]
**Describe the bug**

When running `sanity deploy` from Gitlab pipeline, `Error: Cannot find module '/root/.npm/_npx/43/lib/node_modules/sanity/placeholder.js'` is thrown.

`sanity deploy` works fine when run from local instance, just not in Gitlab pipeline. 

**To Reproduce**

Push Sanity project to Gitlab.
Configure `.gitlab-ci.yml` as:

```
deploy:
    stage: deploy
    image: node:lts
    script:
        - npm install
        - npx sanity install
        - npx sanity deploy
    only:
        - master
```
Push to master
Job results in:
```
$ npx sanity install
internal/modules/cjs/loader.js:883
  throw err;
  ^
Error: Cannot find module '/root/.npm/_npx/43/lib/node_modules/sanity/placeholder.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sanity@2.2.6 postinstall: `node placeholder.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sanity@2.2.6 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-04-16T19_50_20_776Z-debug.log
Install for [ 'sanity@latest' ] failed with code 1
```

**Expected behavior**

Sanity Studio is deployed.

**Which versions of Sanity are you using?**

```
@sanity/cli             2.8.0 (up to date)
@sanity/base            2.8.0 (up to date)
@sanity/components      2.2.6 (up to date)
@sanity/core            2.8.0 (up to date)
@sanity/default-layout  2.8.0 (up to date)
@sanity/default-login   2.8.0 (up to date)
@sanity/desk-tool       2.8.0 (up to date)
@sanity/vision          2.8.0 (up to date)
```

**Which versions of Node.js / npm are you running?**

As is detailed in `.gitlab-ci.yml`: `node:lts`

**Comment by paulThorsen** [comment]
I was able to roll-back to `2.7.4` with no other code changes and the pipeline succeeds and the studio is deployed.

**Comment by paulThorsen** [comment]
Closing as it looks like this got fixed - working on `2.9.1`
